content/applications/hr/time_off.rst

```diff
@@ -0,0 +1,634 @@
+   no requests with a status of :guilabel:`To Submit`, that status option does not appear in the
+   left hand side.
+
+   All departments for the user's employees appear in the list. If there are no allocation requests
+   that fall under that department matching the pre-configured filters, then the list is blank. It
+   is always possible to remove any of the pre-configured filters. Click the X on the specific
+   filter to remove it.
+
+The status column displays the status of each request, with the status highlighted in a specific
+color. The :guilabel:`To Approve` requests are highlighted in yellow, :guilabel:`Approved` requests
+are highlighted in green, :guilabel:`To Submit` (drafts) requests are highlighted in blue, and the
+:guilabel:`Refused` requests are highlighted in gray.
+
+To approve an allocation request, click :guilabel:`✔ Validate` at the end of the line, to refuse a
+request, click :guilabel:`X Refuse`.
+
+.. image:: time_off/allocations.png
+   :align: center
+   :alt: Allocations with the filter, groupings, and status sections highlighted.
+
+If more details are needed, click anywhere on the allocation request line (except for :guilabel:`✔
+Validate` and :guilabel:`X Refuse`) to view the request in detail. The allocation request form
+loads. Depending on the rights of the user, changes can be made. To modify the request, click the
+:guilabel:`Edit` button, make any desired changes, then click :guilabel:`Save`. It is also possible
+to approve or refuse the request from this form. Click the :guilabel:`Validate` button to approve,
+or the :guilabel:`Refuse` button to refuse the request.
+
+.. _time_off/approve-time-off:
+
+Approve time off
+----------------
+
+To view time off requests that need approval, navigate to :menuselection:`Time Off application -->
+Approvals --> Time Off`. The only time off requests visible on this list are for employees the user
+has either :guilabel:`Time Off Officer` or :guilabel:`Administrator` access rights for the *Time
+Off* application.
+
+The default filters in the :guilabel:`Time Off` list are :guilabel:`To Approve`, :guilabel:`My
+Team`, :guilabel:`Active Employee`, and :guilabel:`Active Time Off`. This presents only time off
+requests that need to be approved for current employees on the user's team, for requests that are
+active and not in a draft mode.
+
+The left side of the screen has various grouping options to narrow down the presented time off
+requests. Since only time off requests that need to be approved are shown, the only status options
+are :guilabel:`All` and :guilabel:`To Approve`. To view requests with other statuses, first remove
+the :guilabel:`To Approve` filter. Click the X on the :guilabel:`To Approve` filter to remove it.
+
+To display time off requests for specific departments, click on the department on the left hand
+side. Only requests within the selected department will be presented.
```

Comment:
Adding list headings here too, as well as removing the future tense
```suggestion
The left side of the screen has various grouping options to narrow down the presented time off
requests. Since only time off requests that need to be approved are shown, the only
:guilabel:`STATUS` options are :guilabel:`All` and :guilabel:`To Approve`. To view requests with
other statuses, first remove the :guilabel:`To Approve` filter. Click the :guilabel:`X (remove)`
icon on the :guilabel:`To Approve` filter to remove it.

To display time off requests for specific departments, click on the department on the left hand
side in the :guilabel:`DEPARTMENT` list. Only requests within the selected department are
presented.
```